showpass/Showpass-E2E-tests
# TA-101 Verify guest info forms for each ticket added to cart during checkout

**Description:**
On the event set up page, one can select the following check-box ‘Require guest information for each ticket’. This feature is enabling the attendee personal info form for each ticket added to cart.
We will be verifying that a correct number of forms show up once 1/2/3 or more tickets are added to cart.

- Navigate to [Automation-event-restrictions-101120231100 - 155 Wellington Street West - Toronto - Nov 10, 2023](https://beta.showpass.com/automation-event-restrictions-101120231100/) 
- Add 3 tickets to cart and proceed with the checkout
- Ensure that 3 ‘guest info’ forms show up during checkout

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-101

**How to test:**
Run the following spec and ensure all tests pass: cypress/e2e/checkout/checkout.cy.js

**Test execution results:**

[Cypress Cloud results](https://cloud.cypress.io/projects/e9axi7/runs/182/test-results)
